src/pages/ItemList/components/product.scss

```
@@ -0,0 +1,32 @@
+@import '../../../styles/variables';
+
+.product {
+  @include widthHeightSort(328px, 328px);
+
+  .productImage {
+    @include widthHeightSort(328px, 328px);
+    position: relative;
+    border-radius: 5px;
+
+    .cartImage {
+      @include widthHeightSort(50px, 50px);
+      @include flexSort(null, center, center);
+      @include positionSort(null, 10px, 10px, null);
+      border-radius: 50%;
+      opacity: 0.8;
+      background-color: gray;
+      color: white;
+    }
+  }
+
+  .productName {
+    font-size: 18px;
+  }
+  .productPrice {
+    font-size: 18px;
+    font-weight: bolder;
+  }
+  .productDesc {
+    font-size: 15px;
+  }
```

```suggestion
  .productName {
    font-size: 18px;
  }
  
  .productPrice {
    font-size: 18px;
    font-weight: bolder;
  }
  
  .productDesc {
    font-size: 15px;
  }
```

각 선택자 별로 가독성을 위해서 한칸씩 띄어서 작성해주세요!